content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -39,6 +39,91 @@ model User {
 }
 ```
 
+<SwitchTech technologies={['*', 'postgresql']}>
+
+To map your data model to the database schema, you need to use the `prisma migrate` CLI commands:
+
+```terminal
+npx prisma migrate dev --name init
+```
+
+This command does two things:
+
+1. It creates a new SQL migration file for this migration
+1. It runs the SQL migration file against the database
+
+> **Note**: `generate` is called under the hood by default, after running `prisma migrate dev`. If the `prisma-client-js` generator is defined in your schema, this will check if `@prisma/client` is installed and install it if it's missing.
+
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'mysql']}>
+
+To map your data model to the database schema, you need to use the `prisma migrate` CLI commands:
+
+```terminal
+npx prisma migrate dev --name init
+```
+
+This command does two things:
+
+1. It creates a new SQL migration file for this migration
+1. It runs the SQL migration file against the database
+
+> **Note**: `generate` is called under the hood by default, after running `prisma migrate dev`. If the `prisma-client-js` generator is defined in your schema, this will check if `@prisma/client` is installed and install it if it's missing.
+
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'planetscale']}>
+
+To use Prisma Migrate with PlanetScale, you will also need to add the [`referentialIntegrity` preview feature](/concepts/components/prisma-schema/relations/referential-integrity) to your `schema.prisma` file:
+
+```prisma file=schema.prisma highlight=3;add|9;add
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["referentialIntegrity"]
+}
+
+datasource db {
+  provider             = "mysql"
+  url                  = env("DATABASE_URL")
+  referentialIntegrity = "prisma"
+}
+```
+
+You will then need to connect to your PlanetScale database server using the [PlanetScale CLI](https://docs.planetscale.com/tutorials/automatic-prisma-migrations#:~:text=Install%20the-,PlanetScale%20CLI,-.). First, authenticate with the following command:
+
+```terminal
+pscale auth login
+```
+
+Then create a new database. For example, the following command will create a database with the name `prisma-test`:
+
+```terminal
+pscale db create prisma-test
+```
+
+Create a `develop` branch for your `prisma-test` database:
+
+```terminal
+pscale branch create prisma-test develop
+```
+
+Now connect to this `develop` branch:
+
+```terminal
+pscale connect prisma-test develop
+```
```

Comment:
This is only relevant when using the PlanetScale CLI. When you used a connection string from the PlanetScale UI in the "Connect your database" step, this is not necessary.

So maybe this also should go on the "Connect your database" page as a sub explanation on how to use a "local" database with the `localhost` connection string.

Reply:
I'm slightly confused about how branching works when you use the PlanetScale connection string rather than localhost - do you instead connect to e.g. the `develop` branch by selecting that option in the PlanetScale UI?